index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Potfolio page</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+    href="https://font.googleapis.com/css2?family=Poppin"rel="stylesheet"/>
+    <link
+    href="https://font.googleapis.com/css2?family=Poppins:wght@400;500;600;700;800;900&display=swap"rel="stylesheet"
+    />
+</head>
+<body>
+    <header class="header">
+        <nav class="header_nav">
+            <div class="header_logo"><a class="blue" href="#">My logo</a></div>
+            <div class="header_menu">
+                <img class="menu_icon" src="images/Icon - Menu.png" alt="menu" />
+            </div>
+        </nav>
+    </header>
+  <main>
+    <section class="headline">
```

Comment:
- [ ] Good job adding the background image @lopezivan1216, Yet It seems it could use more alignment perfection. Fixing this will help your website look great.
- [ ] Also, as you can see your fonts are different from the ones in the Figma design. The main aim of this exercise is to help you learn to follow a web design accordingly. `You can click on the text in figma and look through its design detail (font-type, color-code, and more)`
## Figma design

![figma background](https://user-images.githubusercontent.com/69077061/199954293-4a03ac5b-583c-4c61-8d14-fcf052a8ea4f.png)

## Your design

![your background](https://user-images.githubusercontent.com/69077061/199954299-c847224f-a0c8-47a7-812d-a1b2ac352016.png)

Reply:
> * [ ]  Good job adding the background image @lopezivan1216, Yet It seems it could use more alignment perfection. Fixing this will help your website look great.
> * [ ]  Also, as you can see your fonts are different from the ones in the Figma design. The main aim of this exercise is to help you learn to follow a web design accordingly. `You can click on the text in figma and look through its design detail (font-type, color-code, and more)`
> 
> ## Figma design
> ![figma background](https://user-images.githubusercontent.com/69077061/199954293-4a03ac5b-583c-4c61-8d14-fcf052a8ea4f.png)
> 
> ## Your design
> ![your background](https://user-images.githubusercontent.com/69077061/199954299-c847224f-a0c8-47a7-812d-a1b2ac352016.png)

- [ ] As pointed out by the previous reviewer, your background image is still not taking the mobile full width. Kindly widen the background image.
### Your work
![image](https://user-images.githubusercontent.com/86200356/212005214-25117835-1cb1-4ab8-b386-a3c229184278.png)

### Figma
![image](https://user-images.githubusercontent.com/86200356/212005478-d7280e5b-3103-4a24-8dd6-514dec61e591.png)